### ISSUE TITLE
Request classification results

### ISSUE THREAD

**Issue by fkxssaa** [wzmsltw/BSN-boundary-sensitive-network#18]
Thank you for your contribution! I've learned a lot from your code. The caffe files about anet_16 and UntrimmedNet can't compiled on my computer. Could you please send me the classification results of anet and UntrimmedNet on ActivityNet and THUMOS data sets respectively?
email : 349249680@qq.com

**Comment by wzmsltw** [comment]
Hi @fkxssaa  , we obtain [44]'s result files by e-mail in last year. Recently, I found that these result files may have some problem. Thus we are replacing it with classification results of another solution (CUHK at ActivityNet 2017), and will update our arxiv paper in several days.
If you also need their classification results for action detection, you can send a email to [CUHK at ActivityNet 2017].
You can download untrimmednet results on THUMOS-14 testing set on:
https://drive.google.com/file/d/1mfotX6cd8UyO3oMj8ADh4wniywGHtcLp/view?usp=sharing

**Comment by fkxssaa** [comment]
Thank you for your patient help. 
1, I used BSN to generate candidate proposals on THUMOS14. After soft-nms, I only got one percentage point difference between the results and the paper on temporal action proposals task. 
2, Then I used the soft-NMS results, directly taking the first 200 results, and combined them with the Unet classification result top2 to get 400 results, but I got them on The official THUMOS14 evalkit, mAP@0.5=22.4
PS: Because each video is divided into 101 categories, the top2 I took in the UNET results is included in the 20 categories of temporal action detection.I ignored the other categories.
